connectors/sources/salesforce.py

```diff
@@ -1340,10 +1352,26 @@ def get_default_configuration(cls):
                 "tooltip": "The client secret for your OAuth2-enabled connected app. Also called 'consumer secret'",
                 "type": "str",
             },
+            "standard_objects_to_sync": {
+                "display": "textarea",
+                "label": "Standard Objects to Sync",
+                "order": 4,
+                "tooltip": "The list of Salesforce Standard Objects to sync",
+                "type": "list",
+                "value": WILDCARD,
```

Comment:
I know I suggested wildcard before :D .... but maybe we should be more explicit about this ? Given that the list is short maybe we should just prepopulate with everything. It's easier to delete one entry than to build the list yourself
```suggestion
                "value": ",".join(STANDARD_SOBJECTS), 
```

Reply:
Yeah that makes sense especially since it's such a short list 🫡